script/build.js

```diff
@@ -422,7 +422,12 @@ if (options.watch) {
     historyApiFallback: {
       rewrites: [
         { from: '^/track-claims(.*)', to: '/track-claims/' },
-        { from: '^/education/apply-for-education-benefits/application(.*)', to: '/education/apply-for-education-benefits/application/' },
+        { from: '^/education/apply-for-education-benefits/application/1990(\/.*)?', to: '/education/apply-for-education-benefits/application/1990/' }, // eslint-disable-line no-useless-escape
```

Comment:
This doesn't work for me, I still get 1990N redirected to the 1990 form. What does work for me is moving this line below the other two 1990 forms and changing it to 

`{ from: '^/education/apply-for-education-benefits/application/1990(.*)', to: '/education/apply-for-education-benefits/application/1990/' }`

That lets the other two 1990 forms be matched first.

Reply:
Great catch and idea @jbalboni, changes made.